integrations/docker/images/chip-build-bouffalolab/setup.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+set -x
+CURRENT_DIR=$(
+    cd "$(dirname "$0")"
+    pwd
+)
+SDK_ROOT_DIR=$CURRENT_DIR/../
```

Comment:
you run this from `tmp` so SDK_ROOT_DIR will be `/`? Is this intended?

Should we maybe instead pass in the sdk as an argument to make the script easier to read? Then we could use /opt/sdk or similar as sdk paths.

Reply:
Script setup.sh is copied from the repo of Bouffalo Lab IOT SDK, and creates a symbolic link to installed toolchain under the repo for Bouffalo Lab examples building. SDK_ROOT_DIR is root folder of that repo. 

To be clear that TOOLCHAIN_SYMBOLIC_LINK_PATH is added for the path of symbolic link, and this symbolic link will not be created during Docker image building. Now, setup.sh will only download toolchain zip files and extract to /opt/bouffalolab_sdk.